### PR TITLE
Better crosshair init

### PR DESCRIPTION
**Pull request by islemaster**
The visualization overlay was failing to initialize properly when the visualization is less than full-size due to responsive layout.  This was an ordering problem - we are calculating the initial transform matrix during the initial render (which happens in React) but the responsive layout was managed by adding a `'response'` class to the visualization after the first render - thus the initial transform is calculated against the wrong visualization size.

Moves application of that `'responsive'` class into the React render to solve the ordering problem.

### CODE REVIEW

**Comment by islemaster** [comment]
Before: If your window is small and you load GameLab, the crosshair works incorrectly.
After: The crosshair works correctly.

File: apps/src/templates/ProtectedVisualizationDiv.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+import {connect} from 'react-redux';
+import ProtectedStatefulDiv from './ProtectedStatefulDiv';
+
+export const VISUALIZATION_DIV_ID = 'visualization';
+
+export function isResponsiveFromState(state) {
```
Comment (Bjvanminnen) [inline apps/src/templates/ProtectedVisualizationDiv.jsx:14]: Probably deserves a short comment

File: apps/src/templates/ProtectedVisualizationDiv.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+const ProtectedVisualizationDiv = React.createClass({
+  propTypes: {
+    isResponsive: React.PropTypes.bool.isRequired
+  },
+
+  render() {
+    return (
+      <ProtectedStatefulDiv
+          id={VISUALIZATION_DIV_ID}
+          className={this.props.isResponsive ? 'responsive' : ''}
```
Comment (Bjvanminnen) [inline apps/src/templates/ProtectedVisualizationDiv.jsx]: Could do `className={classNames({responsive: this.props.isResponsive})}`, though I'm not sure there's a ton of value when it's just one class like this.

**Comment by Bjvanminnen** [comment]
lgtm. nice cleanup :)

**Pull request merged**